.github/workflows/rubocop.yaml

```diff
@@ -0,0 +1,27 @@
+name: Rubocop
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 2.7
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7
```

Comment:
Confident much? xD
I feel it's pretty new ruby and foreman is not ready at all :thinking:

Reply:
I am running my dev setup on 2.7 for quite some time now. Granted I see a lot of warnings but 2.7.2 dropped these I think.